src/current/v24.1/work-with-virtual-clusters.md

```diff
@@ -114,7 +114,7 @@ sql_txn_commit_count{tenant="demo"} 0
 
 When connected to a virtual cluster from the DB Console:
 
-- Most pages and views are scoped to a virtual cluster. By default the DB Console displays only metrics about that virtual cluster, and excludes metrics for other virtual clusters and the system virtual cluster. To allow the DB Console to display system-level metrics from within a virtual cluster, you can grant the virtual cluster the `can_view_node_info` permission.
+- Most pages and views are scoped to a virtual cluster. The DB Console displays only metrics about that virtual cluster, and excludes metrics for other virtual clusters and the system virtual cluster.
```

Comment:
Thanks for fixing the bullet mistake. Is there any reason this needs to be slightly different from in the overview?

Reply:
@mdlinville can you clarify what's different from overview and I can get the text in sync?

Reply:
Lines 82-3 of cluster-virtualization-overview.md nearly but not 💯 duplicate lines 117 and 118. I'm wondering if we can bring them more in sync.